tests/test_routes.py

```diff
@@ -205,6 +205,7 @@ def test_delete_wishlist(self):
         self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
         # make sure the wishlist is deleted
         response = self.app.get(f"{BASE_URL}/{wishlist.id}/items")
+        print(response.get_json())
```

Comment:
Get rid of the print statement please, thanks

Reply:
removed it!